mautrix_telegram/__main__.py

```diff
@@ -146,6 +144,10 @@ async def start(self) -> None:
                 self.log.error(f"Failed to start bot: {e}")
                 METRIC_BOT_STARTUP_OK.set(0)
 
+        # Explicitly not a startup_action, as startup_actions block startup
+        if self.config["bridge.limits.enable_activity_tracking"] is not False:
```

Comment:
`not False`. Coming from other languages I assume you're want this to run when the setting is `undefined` or `True`.

Reply:
It is indeed unnecessarily specific, a regular `if foo:` would be sufficient. Fixed in e95de09, thanks!